tests/Doctrine/Tests/DBAL/Schema/PostgreSQLSchemaManagerTest.php

```diff
@@ -32,37 +36,35 @@ protected function setUp()
 
     /**
      * @group DBAL-474
+     * @dataProvider filtersSequencesDataProvider
      */
-    public function testFiltersSequences()
+    public function testFiltersSequences(string $platformClass, array $sequencesData, int $fetchAssocExpectationCount) : void
```

Comment:
This test is hard to maintain and not really useful:

1. It used to pass on PostgreSQL 10 before the issue was fixed.
2. It started failing and had to be reworked after the issue was fixed.
3. It enforces implementation details which are not necessarily correct (`fetchAssoc` vs. `fetchAll`).

Let's replace this test with a functional one in `SchemaManagerFunctionalTestCase`. The test should create a sequence using the `AbstractSchemaManager` manager API, and then list the sequences to see if it's created and introspected correctly. This test will not depend on the implementation details of a given platform and will prove that the functionality is portable between the platforms.